platform/viewer/package.json

```diff
@@ -88,10 +89,13 @@
     "vtk.js": "^11.7.2"
   },
   "devDependencies": {
+    "@testing-library/react-hooks": "^3.2.1",
     "cypress": "^3.8.0",
     "gh-pages": "2.0.1",
     "identity-obj-proxy": "3.0.x",
+    "jest-mock-axios": "^3.2.0",
     "lodash": "4.17.15",
-    "lodash.clonedeep": "4.5.0"
+    "lodash.clonedeep": "4.5.0",
+    "react-test-renderer": "^16.12.0"
```

Comment:
Are we using this package?

Reply:
Yes, because of `@testing-library/react-hooks`. It uses `react-test-renderer` but it doesn't come bundled together to allow us to install a specific version of `react-test-renderer`.